workflow/scripts/de_analysis.py

```diff
@@ -127,7 +128,10 @@
 # delete rows, which do not meet our p-value criterion
 # the comparison operator is >= because we drop all values >= our desired alpha
 normalized.drop(normalized[cutoff[1] >= snakemake.config["alpha"]].index, inplace=True)
-# through away these columns
+# copy normalized to get output matrix of genes with lfc and pvalue
+de_genes = normalized.copy()
```

Comment:
Why do we need this copy again? And write it out to a file?

In the end, we fall back to the previous data-frame, anyways.

Reply:
resolved by https://github.com/snakemake-workflows/transcriptome-differential-expression/pull/83/commits/bddf59f81f562588fc963ab7440e4a73ff4e0822